### PR TITLE
Updated UXUI-206 to exist inside of /holidayextras/ui-toolkit/tree/ma…

### PR DESCRIPTION
**Pull request by manifestinteractive**
Updated UXUI-206 to exist inside of /holidayextras/ui-toolkit/tree/master/src/components

Was not able to get Unit Tests to work until told by Jack that Node v0.12.0 had issues with Jest.  Was able to downgrade to v0.10.21 and that got Jest working.  None of the v0.11's I tried worked, so make sure you do not upgrade Node if you are on v0.10.21 otherwise unit tests will no longer work ;)

With v0.10.21, all unit tests are passing.

### CODE REVIEW

**Comment by jodiedoubleday** [comment]
The tests passed fine
Linting fails...
```
src/components/image/__tests__/image-test.js
  13:14  error  Strings must use singlequote  quotes
  14:14  error  Strings must use singlequote  quotes
src/components/image/index.js
  1:1  error  Newline required at end of file but not found  eol-last
```

File: src/components/image/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Tests are written using [Jest](https://facebook.github.io/jest/).
+
+## Parameters
+
+## Example
+
+```
+var React = require('react');
+var ImageComponent = require('ui-component-image');
```
Comment (jodiedoubleday) [inline src/components/image/README.md]: We have removed the `ui-` from all other components, it's not required.

File: src/components/image/code/templates/imageTemplate.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+var React = require('react');
+
+module.exports = function(props) {
+  return (
+    <img alt={props.alt} src={props.src} className="ui-component-image" />
```
Comment (jodiedoubleday) [inline src/components/image/code/templates/imageTemplate.jsx]: We have removed the `ui-` from all other components, it's not required.

**Comment by jodiedoubleday** [comment]
Can you update the docs with an example, and maybe even update the tile to include the image and a child element?

**Comment by manifestinteractive** [comment]
@jodiedoubleday Pushed changes https://github.com/holidayextras/ui-toolkit/commit/5d3bb45e0e70fa6704971128d9353ed1386e0b1d

Also, question on the "can you update the docs with an example" ... are you talking about the README.md file in the component?  That already had the example, which is the same as the other components in format.  Perhaps you are referring to another set of docs?

**Comment by jodiedoubleday** [comment]
Docs as is http://www.ui-toolkit.com
All these examples are held in `docs/examples` You will need to create and image section and add it to the right hand navigation etc.

**Comment by jodiedoubleday** [comment]
you can also remove `dev` folder and `readme.md` files from the individual components now, they are no longer required as of - https://github.com/holidayextras/ui-toolkit/pull/13

**Comment by cameronviner** [comment]
Taking SE/DEV

**Comment by dancork** [comment]
Taking the IP/Dev review

**Comment by dancork** [comment]
Looks good, as Jodie says docs need updating.

**Comment by cameronviner** [comment]
This looks good to me, once Jodie's comments are addressed ill happily +1 it.

**Comment by manifestinteractive** [comment]
Latest commit has updated docs and removed dev / README files, per request.

**Comment by cameronviner** [comment]
This looks pretty good to me +1

**Comment by dancork** [comment]
:+1: Looks good. @jodiedoubleday might be worth adding more margin/padding between the article tags on the doc. They're a bit crammed together.

**Comment by jodiedoubleday** [comment]
@dancork noted, ill be sorting that out in a separate PR

**Comment by jodiedoubleday** [comment]
Look good :+1: (going to update tile in separate PR)

**Pull request merged**